src/sqlight.gleam

```diff
@@ -6,6 +6,16 @@ import gleam/string
 
 pub type Connection
 
+type PreparedStatement
+
+pub opaque type Statement(t) {
```

Comment:
Could we call this `PreparedStatement` to make it more clear to the users please 🙏 

The other is internal so I don't mind the name so much.